### ISSUE TITLE
Accessibility Issues:  Missing form label, Empty form label

### ISSUE THREAD

**Issue by libmanuk** [uklibraries/UKL_HeaderFooter#77]
Update Springshare search form in ukltophdr_sitehdr HTML chunk.
